src/table.js

```diff
@@ -277,9 +277,9 @@ export const addColumnAt = columnIndex => tr => {
   return tr;
 };
 
-// :: (rowIndex: number, clonePreviousRow?: boolean) → (tr: Transaction) → Transaction
-// Returns a new transaction that adds a new row at index `rowIndex`. Optionally clone the previous row.
-//
+// :: (rowIndex: number, cloneIndex?: boolean | number) → (tr: Transaction) → Transaction
```

Comment:
I think you should use union here, this line fails the build

Reply:
try cloneIndex: ?union< boolean, number>

Reply:
Done